pkg/base/helm_test.go

```diff
@@ -67,6 +67,205 @@ func Test_checkChartForVersion(t *testing.T) {
+					HelmVersion: "v3",
+					Namespace:   "test-two",
+				},
+			},
+			want: &Base{
+				Files: []BaseFile{
+					{
+						Path: "deploy-1.yaml",
+						Content: []byte("# Source: test-chart/templates/deploy-1.yaml\nfile: 1\n" +
+							"metadata:\n  name: deploy-1\n  namespace: test-one"),
+					},
+					{
+						Path:    "deploy-2.yaml",
+						Content: []byte("file: 2\nmetadata:\n  name: deploy-2\n  namespace: test-two\n"),
+					},
+				},
+				AdditionalFiles: []BaseFile{
+					{
+						Path:    "Chart.yaml",
+						Content: []byte("name: test-chart\nversion: 0.1.0"),
+					},
+				},
+			},
+		},
+		{
+			name: "helm v2 namespace insertion",
+			args: args{
+				upstream: &upstreamtypes.Upstream{
+					Name: "namespace-test",
+					Files: []upstreamtypes.UpstreamFile{
+						{
+							Path:    "Chart.yaml",
+							Content: []byte("name: test-chart\nversion: 0.1.0"),
+						},
+						{
+							Path:    "templates/deploy-1.yaml",
+							Content: []byte("metadata:\n  name: deploy-1\n  namespace: test-one"),
+						},
+						{
+							Path:    "templates/deploy-2.yaml",
+							Content: []byte("metadata:\n  name: deploy-2"),
+						},
+					},
+				},
+				renderOptions: &RenderOptions{
+					HelmVersion: "v2",
+					Namespace:   "test-two",
+				},
+			},
+			want: &Base{
+				Files: []BaseFile{
+					{
+						Path:    "deploy-2.yaml",
+						Content: []byte("metadata:\n  name: deploy-2\n  namespace: test-two\n"),
+					},
+					{
+						Path:    "deploy-1.yaml",
+						Content: []byte("metadata:\n  name: deploy-1\n  namespace: test-one"),
+					},
+				},
+				AdditionalFiles: []BaseFile{
+					{
+						Path:    "Chart.yaml",
+						Content: []byte("name: test-chart\nversion: 0.1.0"),
+					},
+				},
+			},
+		},
+		{
+			name: "helm v3 namespace insertion with multidoc",
+			args: args{
+				upstream: &upstreamtypes.Upstream{
+					Name: "namespace-test",
+					Files: []upstreamtypes.UpstreamFile{
+						{
+							Path:    "Chart.yaml",
+							Content: []byte("name: test-chart\nversion: 0.1.0"),
+						},
+						{
+							Path: "templates/deploy.yaml",
+							Content: []byte("metadata:\n  name: deploy-1\n  namespace: test-one\n" +
+								"---\n" +
+								"metadata:\n  name: deploy-2\n  namespace: test-two"),
+						},
+					},
+				},
+				renderOptions: &RenderOptions{
+					HelmVersion: "v3",
+					Namespace:   "test-two",
+				},
+			},
+			want: &Base{
+				Files: []BaseFile{
+					{
+						Path: "deploy.yaml",
+						Content: []byte("# Source: test-chart/templates/deploy.yaml\n" +
+							"metadata:\n  name: deploy-1\n  namespace: test-one\n" +
+							"---\n" +
+							"# Source: test-chart/templates/deploy.yaml\n" +
+							"metadata:\n  name: deploy-2\n  namespace: test-two"),
+					},
+				},
+				AdditionalFiles: []BaseFile{
+					{
+						Path:    "Chart.yaml",
+						Content: []byte("name: test-chart\nversion: 0.1.0"),
+					},
+				},
+			},
+		},
+		{
+			name: "helm v3 namespace insertion with multidoc",
```

Comment:
nit: this has the same name as the previous test

Reply:
copy/paste, good catch!